From 93f6ed27f8c796cf087efda2c0ec0a25eab0acbd Mon Sep 17 00:00:00 2001
From: Adrien Destugues <pulkomandy@gmail.com>
Date: Sat, 25 Apr 2015 16:58:40 +0200
Subject: Restore patch for type definitions.

* This was removed when updating from 1.0.0. The lib builds without it,
but then all clients will fail to compile. Please do not remove this
patch again.

diff --git a/ebml/c/libebml_t.h b/ebml/c/libebml_t.h
index 71fe115..8e73ff9 100644
--- a/ebml/c/libebml_t.h
+++ b/ebml/c/libebml_t.h
@@ -72,7 +72,7 @@ extern "C" {
     typedef uint16_t uint16;
     typedef uint8_t uint8;
 # endif // __GNUC__
-#elif defined(__BEOS__)
+#elif defined(__BEOS__) || defined(__HAIKU__)
 #include <SupportDefs.h>
 #elif defined(DJGPP)        /* SL : DJGPP doesn't support POSIX types ???? */
     typedef signed long long int64;
@@ -98,7 +98,7 @@ extern "C" {
     typedef uint32_t uint32;
     typedef uint16_t uint16;
     typedef uint8_t uint8;
-#elif defined(__BEOS__)
+#elif defined(__BEOS__) || defined(__HAIKU__)
 # include <support/SupportDefs.h>
 #else // anything else (Linux, BSD, ...)
 # include <inttypes.h>
-- 
2.2.2

